spring-boot-project/spring-boot-actuator/src/main/java/org/springframework/boot/actuate/metrics/MetricsEndpoint.java

```diff
@@ -112,15 +112,20 @@ private Tag parseTag(String tag) {
 		return Tag.of(parts[0], parts[1]);
 	}
 
-	private void collectMeters(List<Meter> meters, MeterRegistry registry, String name,
-			Iterable<Tag> tags) {
+	private boolean findFirstMatchingMeters(List<Meter> meters, MeterRegistry registry,
```

Comment:
Rather than modifying a parameter, this method could simply return a `Collection`.